liteflow-labs/liteflow-js
# Select Component: Added a default max height and a property to set a max height

### Project organization

- Trello: https://trello.com/c/D8qc96P4
- Related https://github.com/liteflow-labs/hodooi/issues/29 (Need to create a PR that updates the libraries to fix the problem)

### Description

This PR's goal is to add more control on the `Select` component dropdown height. I added a default height of Chakra `52` => `13rem` as well as the property `dropdownMaxHeight` to allow passing of a custom max height. This solves problems similar to: https://github.com/liteflow-labs/hodooi/issues/29 

### How to test

Storybook: https://storybook-components-kqgkx0r1u-liteflow.vercel.app/?path=/story/templates-select-select--with-many-choices-and-max-height

See the newly created `dropdownMaxHeight` parameter in the controls section, change or remove to see default, property in action.

### Checklist
